docs/integrations/databases/opentelemetry/memcached-opentelemetry.md

```diff
@@ -163,14 +167,19 @@ import LogsOutro from '../../../reuse/apps/opentelemetry/send-logs-outro.md';
 <LogsOutro/>
 
 
-## Sample log messages
+## Sample log message
```

Comment:
These are standard headers throughout all apps docs that we keep plural by default (even if there's only one example)

```suggestion
## Sample log messages
```